### ISSUE TITLE
Remove shebang

### ISSUE THREAD

**Issue by fabaff** [maaaaz/androwarn#26]
Please remove the shebang from all files which don't need it.

For the Fedora package I have to:

```bash
sed -i -e '/^#!\//, 1d' androwarn/{__init__.py,androwarn.py,warn/*/*.py,warn/*/*/*.py}
```

**Comment by maaaaz** [comment]
Hello @fabaff,
Thanks for reporting, but are you sure that a shebang is not needed in `androwarn.py` ? 

It is the entry point, I doubt that it's not a good practice to specify it.
And overall, it is not a [good practice](https://www.python.org/dev/peps/pep-0394/#recommendation) to specify in each file, even if they are not directly called ?

Cheers.

**Comment by fabaff** [comment]
> Thanks for reporting, but are you sure that a shebang is not needed in androwarn.py ?

Yes, I am. `androwarn` will contain the shebang as this is the entry point for an installation. But if want to allow users to run it from a checkout then the shebang might be needed in `androwarn.py`.

> And overall, it is not a good practice to specify in each file, even if they are not directly called ?

Those files never get called from the command-line thus not need for shebangs.

**Comment by maaaaz** [comment]
> Thanks for reporting, but are you sure that a shebang is not needed in androwarn.py ?
>> Yes, I am. `androwarn` will contain the shebang as this is the entry point for an installation. But if want to allow users to run it from a checkout then the shebang might be needed in `androwarn.py`.  

Ok that's clear, indeed it is expected to also be able to run directly `androwarn.py` so this will be the only file with a shebang.
 
> And overall, it is not a good practice to specify in each file, even if they are not directly called ?
>> Those files never get called from the command-line thus not need for shebangs.

Clear
